src/app/views/users/UsersController.jsx

```diff
@@ -124,7 +124,7 @@ export class UsersController extends Component {
             }
             notifications.add(notification);
             console.error("Error mapping users to state: ", error);
-            log.add(eventTypes.unexpectedRuntimeError, {message: `Error mapping users to state:\n${JSON.stringify(error)}`});
+            log.event(`Error mapping users to state:\n${JSON.stringify(error)}`, log.error(error));
```

Comment:
no need to include the error in the event name since it's also added by `log.error`